eng/illink.targets

```
@@ -35,14 +36,16 @@
     <ILLinkTrimInputSymbols>$(ILLinkTrimInputPath)$(TargetName).pdb</ILLinkTrimInputSymbols>
     <ILLinkTrimOutputPath>$(IntermediateOutputPath)</ILLinkTrimOutputPath>
 
-    <ILLinkTrimXml Condition="'$(ILLinkTrimXml)' == '' and Exists('$(MSBuildProjectDirectory)/ILLinkTrim.xml')">$(MSBuildProjectDirectory)/ILLinkTrim.xml</ILLinkTrimXml>
+    <ILLinkTrimXml Condition="'$(ILLinkTrimXml)' == '' and Exists('$(ILLinkDirectory)ILLinkTrim.xml')">$(ILLinkDirectory)ILLinkTrim.xml</ILLinkTrimXml>
     <!-- ILLinkTrim_LibraryBuild.xml files are only used during building the library, not an app. They shouldn't be embedded into the assembly. -->
-    <ILLinkTrimXmlLibraryBuild Condition="'$(ILLinkTrimXmlLibraryBuild)' == '' and Exists('$(MSBuildProjectDirectory)/ILLinkTrim_LibraryBuild.xml')">$(MSBuildProjectDirectory)/ILLinkTrim_LibraryBuild.xml</ILLinkTrimXmlLibraryBuild>
+    <ILLinkTrimXmlLibraryBuild Condition="'$(ILLinkTrimXmlLibraryBuild)' == '' and Exists('$(ILLinkDirectory)ILLinkTrim_LibraryBuild.xml')">$(ILLinkDirectory)ILLinkTrim_LibraryBuild.xml</ILLinkTrimXmlLibraryBuild>
     <ILLinkDescriptorsXmlIntermediatePath>$(IntermediateOutputPath)ILLink.Descriptors.xml</ILLinkDescriptorsXmlIntermediatePath>
 
     <ILLinkSubstitutionsXmlIntermediatePath>$(IntermediateOutputPath)ILLink.Substitutions.xml</ILLinkSubstitutionsXmlIntermediatePath>
     <ILLinkLinkAttributesXmlIntermediatePath>$(IntermediateOutputPath)ILLink.LinkAttributes.xml</ILLinkLinkAttributesXmlIntermediatePath>
 
+    <ILLinkSuppressionsXmlFile>$(ILLinkDirectory)ILLink.Suppressions</ILLinkSuppressionsXmlFile>
```

Why not include the extension here?

```suggestion
    <ILLinkSuppressionsXmlFile>$(ILLinkDirectory)ILLink.Suppressions.xml</ILLinkSuppressionsXmlFile>
```

---

Currently, this is to support having configuration-specific suppression files e.g. 

https://github.com/dotnet/runtime/blob/9bc6f384e9dd7cb0ba843d0964b2e72b5fd7b16b/eng/illink.targets#L48

If I can figure out how to address https://github.com/dotnet/runtime/pull/40691#discussion_r470778907) (meaning that we wouldn't need code like [this](https://github.com/dotnet/runtime/blob/9bc6f384e9dd7cb0ba843d0964b2e72b5fd7b16b/src/libraries/illink-sharedframework.targets#L94-L95)), we can have something like this as well:

`<ILLinkSuppressionsTargetArchitectureSpecificXmlFile>$(ILLinkSuppressionsXmlFile).$(TargetArchitecture)</ILLinkSuppressionsTargetArchitectureSpecificXmlFile>`